README.md

```diff
@@ -1,2 +1,199 @@
-# Budget_App
-Mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- [ ] These two links are broken. Kindly fix them or remove them.

- [ ] Kindly deploy your app and add the link in the README

- [ ] Your video demo length is longer than 5 minutes. Kindly provide video presentation that is less than 5 minutes in length based on the requirements.

Reply:
Fixed